app/src/main/java/com/anytypeio/anytype/ui/sets/ObjectSetFragment.kt

```diff
@@ -1167,6 +1170,16 @@ open class ObjectSetFragment :
         inflater, container, false
     )
 
+    private fun observeSelectingTemplate() {
+        findNavController().currentBackStackEntry?.savedStateHandle?.getLiveData(
+            ARG_TEMPLATE_ID,
+            ""
```

Comment:
it would be great to have a constant with some meaningful naming here

Reply:
fixed